swarooppatilx/scruter
# Add header and footer to Privacy Policy page

# What does this PR do?

This pull request adds a header and footer to the Privacy Policy page, enhancing the user navigation and overall layout of the page. 

**Fixes:** #268

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

To verify the changes, please follow these steps:

1. Navigate to the Privacy Policy page in the application.
2. Check if the header and footer are displayed correctly.
3. Ensure that the header includes the correct links and branding.
4. Verify that the footer includes all relevant information (e.g., copyright, links to other pages).
5. Test responsiveness by resizing the browser window or checking on different devices.

### Additional Tests

- [x] Ensure that the header and footer render correctly on all supported browsers.
- [x] Check for any console errors or warnings related to the header and footer components.
- [x] Verify that the header and footer do not interfere with the existing layout of the Privacy Policy content.

## Mandatory Tasks

- [x] Update any relevant documentation, if applicable.

Uploading footer and header.mp4…
